src/Options.ts

```diff
@@ -110,7 +110,7 @@ export type Options = ReturnType<typeof parseOptions>;
  * Parse and normalize command line arguments passed to the script.
  */
 export function parseOptions() {
-  const argv = yargs.options(options).parseSync();
+  const argv = yargs(process.argv.slice(2)).options(options).parseSync();
```

Comment:
Only got a minor yargs version bump, but somehow it's function signature changed :/